examples/examples.test.ts

```diff
@@ -1,13 +1,20 @@
 import {assert} from 'chai';
 import * as vl from '../src/vl';
 
-const ZSchema = require('z-schema');
+const zSchema = require('z-schema');
 const inspect = require('util').inspect;
 const dl = require('datalib');
 const fs = require('fs');
 const path = require('path');
 
-const validator = new ZSchema({
+zSchema.registerFormat('color', function (str) {
+  return false;
```

Comment:
Hmm, don't you have to actually validate the string? I'm surprised that the tests pass if the validator always returns false.